src/test/resources/alfresco/transforms/0999-testPipeline.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Test context dummy pipeline that allows us to test the availability of transformers with different source sizes. No longer possible with the standard OOTB config.